content/1.7/back-office/login-listing.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Login into Back-Office
+weight: 1
+---
+# LOGIN
+
+[Login UI](static/login.png)
+
+This UI is the main feature to authorize the Prestashop Back-Office administrator, and to log-in to administrate and maintain all the Prestashop Back-Office  features and functionalities. This page contains only one main element - the log in form.
+
+## Top UI Section
+
+- The top section represents the Prestashop main title name and the Prestashop version, that the administrator is accessing.
+
+## Middle UI Section
+
+- **Logo** - the official Preston seabird logo of Prestashop.
+- **Shop name** - the shop name, that was defined during the installation process of Prestashop application.
+- **Email address input** - the specific superadmin profile email input text field. The email is the exact that was defined during the installation process of Prestashop. It must be used at the first log-in attempt. 
+- **Password input** - the specific superadmin profile password input text field. The password is the exact that was defined during the installation process of Prestashop. It must be used at the first log-in attempt. 
+- **Login CTA button** - the button, that executes the logging in action, the user then is instantly redirected to the default main dashboard page.
+- **Stay logged in checkbox** - this checkbox and its function is common in many logging-in submission forms. Once the checkbox is marked, the browser will remember the logging-in status, and the administrator will not be required to input email and password in the next attempt of logging-in. The administrator will be authenticated and redirected to the dashboard instantly.
+- **I forgot my password** - specific link, in case administrator forgets the Prestashop authentication password. Once clicked, the UI instantly flips to the new UI which contains the email input. After submitting the email there, a link will be generated and sent to the defined email.
```

Comment:
What does the email contains? What can you do? Reset the password?

Reply:
updated with the information